src/python/pants/backend/jvm/targets/jvm_target.py

```diff
@@ -74,6 +75,9 @@ def __init__(self,
                               rollbacks, but in certain cases may conflict with user libraries.
     :type zinc_file_manager: bool
     """
+    deprecated_conditional(lambda: resources is not None, '1.5.0dev0',
```

Comment:
i think so, also https://github.com/pantsbuild/pants/commit/42893403743a399c89fa7371d8f5ec8ed2ab836d just went in.